core/baseline-core-v1.0-psd01.md

```diff
@@ -566,678 +637,1941 @@ Below we list and define the components of each layer. The detailed requirements
+* A BPI Account belonging to the BPI User. Note that a BPI Account as defined in Section 6 is different from a BPI User account as defined in [Section 5.2](#52-BPI-User-Account).
+* A BPI Workgroup as defined in [Section 6](#-6-Agreement-Execution). 
+* A BPI Workflow as defined in [Section 6](#-6-Agreement-Execution).
+* A BPI Workstep as defined in [Section 6](#-6-Agreement-Execution).
+* A BPI Transaction as defined in [Section 6](#-6-Agreement-Execution). 
 
-Note that a security policy consists of authentication and authorization rules for the workgroup participants. Note also that one or more workgroup administrators define the workgroup security policy.
+There may be other BPI components such as role definitions or security policies. The enablement of additional components in a BPI beyond this standard is left to each specific implementation.
 
-**[RXX]** A workgroup MUST have at least one privacy policy.
+**[R79]** An internal BPI User MUST be able to create, read, update and delete all components of a BPI.
 
-A privacy policy the data visibility rules for each participant.
+**[R80]** A BPI User MUST be able to send and receive BPI messages from other BPI users using the BPI Communication capability as defined in [Section 5.3](#53-BPI-Communication).
 
-**[RXX]** A workgroup administrator MUST be able to perform at minimum the following functions:
-* add or remove one or more participants
-* create, update and delete both security and privacy policies.
-* delete or archive a workgroup
+**[R81]** A BPI User MUST be able to create, read BPI messages from other BPI users using the BPI Communication capability as defined in [Section 5.3](#53-BPI-Communication).
 
-**[OXX2]** A workgroup MAY have more than one administrator.
+Note that when a BPI User executes these capabilities they are understood to be enabled by services under the direct custody of the BPI User. However, the BPI User may delegate these capabilities to the BPI the BPI user operates in, called a custodial approach.  
 
-**[CRXX]>[OXX2]** There MUST a consensus model for administrative actions.
+**[R82]** A BPI User MUST be able to delegate one or more BPI User capabilities to a 3rd party that is also a BPI User.
 
-**[OXX3]** A workgroup MAY be attached to one or more workstep instances. 
+## 5.2 BPI User Account
 
-**[OXX4]** A Workgroup MAY be attached to one or more workflow instances.
+A BPI User Account is a key component of a BPI because it is the anchor point for all important BPI functions -- authentication and authorization, messages, workgroups, workflows, worksteps, transactions etc. Without the notion of a cryptographically verifiable state of a BPI User within a BPI which is afforded by a BPI User Account, no other BPI functions will be able to operate successfully since all functions are predicated on cryptographically verifiable counterparties that maintain a referable and verifiable state within a BPI.  
```

Comment:
two missing commas.

Reply:
`- which is afforded by a BPI User Account -`

Reply:
fixed